tests/unittest/tools/test_perf_sanity_matching.py

```
@@ -39,63 +43,239 @@ def _benchmark_data(**overrides: object) -> dict[str, object]:
+    Without this, renaming the yaml key would leave the tests below green while
+    the documented behaviour silently changed.
+    """
+    repo_root = pathlib.Path(__file__).resolve().parents[3]
+    module_path = repo_root / "tests" / "integration" / "defs" / "perf" / "test_perf_sanity.py"
+    tree = ast.parse(module_path.read_text())
+
+    iterations_values = [
+        value
+        for node in ast.walk(tree)
+        if isinstance(node, ast.Dict)
+        for key, value in zip(node.keys, node.values)
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)